centaur/src/main/resources/standardTestCases/wdl_draft3/custom_mount_point/custom_mount_point.wdl

```diff
@@ -0,0 +1,38 @@
+version 1.0
+
+task t {
+  input {
+    String v
+  }
+
+
+  output {
+    String out1 = read_string("some_file")
+    String out2 = read_string("/some/mnt/some_file")
+  }
+  command <<<
+
+    echo "bazqux" > some_file
+
+    cd /some/mnt
+    echo "foobar" > some_file
+
+  >>>
+  runtime {
+    docker: "ubuntu:" + v
+    disks: "local-disk 20 SSD, /some/mnt 20 SSD"
+  }
+
+}
+workflow custom_mount_point {
+
+  call t {
+    input:
+      v = "latest"
+  }
+
+  output {
+    String o2 = t.out2
+    String o1 = t.out1
+  }
+}
```

Comment:
also stowaway?

Reply:
Yep, deliberate (long-winded explanation follows)

---

This is a WDL that I _wanted_ to add as an upgrade test, but it requires PAPI whereas the upgrade test suite only supports local backend.

What's being checked in here is the result of me running the upgrade script manually.

This lets me test the correctness of the upgrade script at this instant in time (no regression protection) and more importantly extra regression coverage for WDL 1.0.